### ISSUE TITLE
Sound playback nodes not compatible with Godot 3

### ISSUE THREAD

**Issue by fleskesvor** [fleskesvor/escoria-in-daiza#1]
To be able to open scenes in Godot 3, nodes of types `SamplePlayer` and `AudioStreamPlayer` needs to be removed/replaced. They are triggered by an `AnimationPlayer` in the same scene, but I think it is safe to let that node remain. The following scenes are affected by this issue:

- globals/bg_music.tscn
- ./scenes/intro-game/bad_bamboo.tscn
- ./scenes/intro-game/good_bamboo.tscn
- ./scenes/second_scene/bad_bamboo_2.tscn
- ./scenes/second_scene/good_bamboo_3.tscn

**Comment by fleskesvor** [comment]
I've added sound back in. The background music was easy, since I just had to add an `AudioStreamPlayer` with the music and set it on autoplay. I've added the "pick up" sound effect bad in the other scenes with `AudioStreamPlayer2D`s, but the best I managed was to enable the "playing" property with the animation player, and then turn it off again with a second key frame once the track is done playing, to prevent it from playing again.
